bin/cli.js

```diff
@@ -158,6 +158,10 @@ if (opts.slow) {
   mocha.slow(opts.slow)
 }
 
+if (opts.require) {
+  require(fs.absolute(opts.require));
+}
+
```

Comment:
Fair. I was just concerned with requiring a single file for myself.
Regarding what happens with multiple `--require` the last will be loaded and that's it.
That being said it would be straightforward to load multiple files similarly to what's done with the `--client-scipts` option which is to pass a comma delimited string. I can make that change if you want.